core/src/main/java/org/infinispan/configuration/cache/ClusteringConfiguration.java

```diff
@@ -19,7 +19,7 @@
          AttributeDefinition.builder("remoteTimeout", TimeUnit.SECONDS.toMillis(15)).build();
 
    static AttributeSet attributeDefinitionSet() {
-      return new AttributeSet(ClusteringConfiguration.class, CACHE_MODE);
+      return new AttributeSet(ClusteringConfiguration.class, CACHE_MODE, REMOTE_TIMEOUT);
```

Comment:
The REMOTE_TIMEOUT attribute in SyncConfiguration is actually a ref to the ClusteringConfiguration one, so @rvansa is right